src/Libraries/DSOffice/DSOffice.csproj

```diff
@@ -13,17 +13,33 @@
   <PropertyGroup>
     <NoWarn>MSB3539;CS1591;NUnit2005;NUnit2007;CS0618;CS0612;CS0672</NoWarn>
   </PropertyGroup>
-    <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
-        <PackageReference Include="Microsoft.Office.Interop.Excel" Version="15.0.4795.1001">
-            <ExcludeAssets>runtime</ExcludeAssets>
-        </PackageReference>
-        <Reference Include="WindowsBase" />
-    </ItemGroup>
-    <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
-        <PackageReference Include="Microsoft.Office.Interop.Excel" Version="15.0.4795.1001" />
-        <PackageReference Include="MicrosoftOfficeCore" Version="15.0.0" />
-    </ItemGroup>
-  <ItemGroup Label ="Common deps">
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
+    <PackageReference Include="Microsoft.Office.Interop.Excel" Version="15.0.4795.1001">
+        <ExcludeAssets>runtime</ExcludeAssets>
+    </PackageReference>
+    <Reference Include="WindowsBase" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
+    <PackageReference Include="Microsoft.Office.Interop.Excel" Version="15.0.4795.1001">
+        <ExcludeAssets>all</ExcludeAssets>
```

Comment:
Do no add as typical reference, just use the path of the package to add the dll as an embedded reference

Reply:
Can you explain again why you're adding it as an embedded reference?

Reply:
I chose embedded types so that we can avoid having to ship the Interop.Excel.dll and office.dll (these were in the GAC for net48)
They might cause conflicts with host integrators